homeassistant/components/webostv/services.yaml

```diff
@@ -39,8 +39,7 @@ command:
     command:
       name: Command
       description: >-
-        Endpoint of the command. Known valid endpoints are listed in
-        https://github.com/TheRealLink/pylgtv/blob/master/pylgtv/endpoints.py
+        Endpoint of the command.
```

Comment:
This is now just a single short line, and should be on the same line as `description:`

Reply:
Adjusted it.